guides/release/components/triggering-changes-with-actions.md

```diff
@@ -7,6 +16,27 @@ But what about the opposite direction? How does data flow back out of
 the component to the parent? In Ember, components use **actions** to
 communicate events and changes.
 
+To make this easier to explain, we need to bring another design pattern
+into the scope.
+
+## Data-Down, Actions-Up
+
+A Data-Down, Actions-Up pattern allows you to abstract your
+component from your application scope and think about it as a wrapper
+with triggers, data, and UI. Think about your component as an empty
+bucket with a valve at the bottom. When you refill it with water you
+are passing Data-Down into it. When you open a valve, you initiate
+Action-Up. Now your component is driven by your actions and uses the
+data it holds. In Ember, components use actions to communicate events
+and changes within your component or up to the parent component.
+
+Now you know that the child component can be used to chain activity
+from user interactions (such as a button click) and propagate that
+action for data flow within its own **actions** or out of the child
+component to the parent component. As we previously stated, recipient
```

Comment:
> As we previously stated

it wasn't that long ago, and I somewhat feel like I'm being accused of poor memory skills. Is there a better way to phrase this?  Maybe something like combining with the previous sentence? Maybe that'd fix the pronoun reference, too?